src/daemons/GraphDaemon.cpp

```diff
@@ -223,6 +227,15 @@ Status setupLogging() {
     return status;
 }
 
+Status initWebService(nebula::WebService* svc) {
+    LOG(INFO) << "Starting Graph HTTP Service";
+    auto& router = svc->router();
+    router.get("/sysinfo").handler([](PathParams&&) {
```

Comment:
Maybe we can move this to src/webservice/WebService.app, there is no need to declare it in every daemon.

Reply:
Good point, Let me try this on